src/app/auth/session-selection/session-selection.component.ts

```diff
@@ -64,6 +64,8 @@ export class SessionSelectionComponent implements OnInit {
     if (repoOrg && repoName) {
       window.localStorage.setItem('org', repoOrg);
       window.localStorage.setItem('dataRepo', repoName);
+    } else {
+      window.localStorage.clear();
```

Comment:
I'm sorry if I wasn't clear in my previous comment, but you should not use `clear()` for the whole local storage. Instead, only delete the items `org` and `dataRepo`. There are other things that are kept in the local storage that should be kept, including the logs.

Reply:
oh, i see haha. i'll make the relevant changes!

Reply:
maybe we should remove the items first before setting it? might be better than an else remove